### ISSUE TITLE
Show Embed Error on AMP for IDT1, and VJ Includes without an AMP version

### ISSUE THREAD

**Issue by jamesdonoh** [bbc/simorgh#5987]
To be completed alongside: https://github.com/bbc/simorgh/issues/7128.

### Is your feature request related to a problem? Please describe.
Some Includes have no AMP version available and currently do not render on this version of our pages. In these cases we should use the `psammead-embed-error` component updated in https://github.com/bbc/psammead/issues/3214.

### Describe the solution you'd like
Implement the business rules in Simorgh for when this component should be shown for certain types of includes - the output from [include classifier](https://github.com/bbc/simorgh/blob/latest/src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/includeClassifier.js) will help us with this.

- The link on the fallback needs to point to the Include in the Canonical version.
  - An `id` field with the value used in this `href` will be provided from: https://github.com/bbc/simorgh/issues/7128.
- Please see design notes for link `text` requirements.
  - Translations will not be added in this PR.

### Designs

https://zpl.io/agY9drR (See notes.)

### A11y

Given I use a screen reader
And I am viewing the AMP fallback
When I navigate to the link to canonical
Then _the link should make sense out of context_

Given I use a screen reader
And I am viewing the AMP fallback
When I activate the link to the content on canonical 
Then I am taken to the include content on canonical
And this content has focus (with a tabindex value of -1 on the container)

Given I am viewing the AMP fallback
When I change my colour preferences in Firefox
And when I am viewing in a custom high contrast mode on PC
Then I should be able to see a box (border) that contains the the AMP fallback

### Describe alternatives you've considered
n/a

#### VJ Includes
- If AMP version exists, render it (see https://github.com/bbc/simorgh/issues/5989).
- Otherwise, show embed error.
#### IDT1 Includes
- Show embed error.

- [x] This feature is expected to need manual testing.
- Asset with supported AMP VJ include: https://www.test.bbc.com/gahuza/23323415.amp

### Additional context
None.

**Comment by simonsinclair** [comment]
Blocked pending https://github.com/bbc/psammead/issues/3214.

**Comment by simonsinclair** [comment]
Unblocked following the merge of https://github.com/bbc/psammead/pull/3573.
